### ISSUE TITLE
Doesn't redirect media content

### ISSUE THREAD

**Issue by berarma** [splitbrain/dokuwiki-plugin-redirect#3]
I've moved some PDF files around namespaces and I've tried redirecting with a line like this:

documents:informes:doc.pdf documents:doc.pdf

It doesn't seem to work.

**Comment by splitbrain** [comment]
Correct. Media redirects are not supported currently. Pull Requests welcome
